src/UniGetUI.Core.Settings/SettingsEngine_Dictionaries.cs

```diff
@@ -66,29 +66,29 @@ public static partial class Settings
             {
                 Logger.Error($"Could not load dictionary name {setting}");
                 Logger.Error(ex);
-                return new();
+                return [];
             }
         }
 
         // Returns an empty dictionary if the setting doesn't exist and null if the types are invalid
-        public static IReadOnlyDictionary<K, V>? GetDictionary<K, V>(string setting)
+        public static IReadOnlyDictionary<K, V?> GetDictionary<K, V>(string setting)
             where K : notnull
         {
-            return _getDictionary<K, V>(setting);
+            return _getDictionary<K, V?>(setting);
         }
 
         public static void SetDictionary<K, V>(string setting, Dictionary<K, V> value)
             where K : notnull
         {
-            dictionarySettings[setting] = value.ToDictionary(
+            _dictionarySettings[setting] = value.ToDictionary(
                 kvp => (object)kvp.Key,
                 kvp => (object?)kvp.Value
             );
 
             var file = Path.Join(CoreData.UniGetUIDataDirectory, $"{setting}.json");
             try
             {
-                if (value.Any()) File.WriteAllText(file, JsonSerializer.Serialize(value));
+                if (value.Count != 0) File.WriteAllText(file, JsonSerializer.Serialize(value));

```

Comment:
Why? .Any() is supposed to be better, right?

Reply:
Performance reason. Count is a property, not a function

Reply:
See [CA1860: Avoid using 'Enumerable.Any()' extension method](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1860)